src/Relation/Binary/Indexed/Homogeneous/Structures.agda

```diff
@@ -20,7 +20,10 @@ module Relation.Binary.Indexed.Homogeneous.Structures
 open import Data.Product using (_,_)
 open import Function.Base using (_⟨_⟩_)
 open import Level using (Level; _⊔_; suc)
-open import Relation.Binary as B using (_⇒_)
+open import Relation.Binary.Core using (_⇒_)
+import Relation.Binary.Definitions as Definitions
+import Relation.Binary.Bundles as Bundles
+import Relation.Binary.Structures as Structures
```

Comment:
I am not very sure about the naming schemes here

Reply:
Can't you import them all as `B`?

Reply:
That worked, thanks @gallais!